dev/integration_tests/ios_add2app/README.md

```diff
@@ -0,0 +1,21 @@
+# iOS Add2App Test
+
+This application demonstrates some basic functionality for Add2App,
+along with a native iOS ViewController as a baseline and to demonstrate
+interaction.
+
+The following functionality is currently implemented:
+
+1. A regular iOS view controller, similar to the default `flutter create` template.
+1. A FlutterViewController subclass that takes over full screen. Demos showing this both from a cold/fresh engine state and a warm engine state.
```

Comment:
Does the subclass do anything? Makes one ask what's the difference between this one and 1.